src/dns/resolver.cpp

```diff
@@ -88,7 +88,7 @@ evdns_base *Resolver::get_evdns_base(void) {
     // By default we don't randomize the query's case
     std::string randomiz{"0"};
     if (settings.find("randomize_case") != settings.end()) {
-        randomiz = settings["randomize_case"];
+        randomiz = settings["randomize_case"].as<std::string>();
```

Comment:
In this case, I think it would be nice for the conversion to happen automatically. Here `randomiz` is clearly a string, so I'd expect that, if you attempt to assign `Item` to string it would either work or throw exception. Does it make sense?